### PR TITLE
[Bug] Rate tiers wont display on plan

### LINKED ISSUE #842: [Bug] Rate tiers won't display on plan card

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #842 

image

image

image

### CODE REVIEW

File: app/components/benefits/plan.hbs [hunk 1]
```diff
@@ -28,60 +28,139 @@
...
-                With wellness modifier of {{format-money @plan.contributionsEmployeeWellnessModifier}}
-              </EmberTooltip>
+          {{#if @plan.ratesAgeTiersEmployee.length}}
+            <a href="#" {{on "click" (prevent-default (toggle "showEmployee" this))}}>
+              {{#if this.showEmployee}}
+                <i class="caret up icon"></i>
+              {{else}}
+                <i class="caret down icon"></i>
+              {{/if}}
+            </a>
```
Comment (James1x0) [inline app/components/benefits/plan.hbs:38]: This looks odd in display. Probably put the arrow to the right of the rate range and just make the "u" clickable.

**Review by James1x0 — CHANGES_REQUESTED** [review]
See comment

**Comment by JavaLavaMT** [comment]
image

File: app/components/benefits/plan.hbs (hunk 1 above)
Comment (JavaLavaMT) [reply app/components/benefits/plan.hbs]: 🏁

**Pull request merged**